src/common/licensing/trial_manager.js

```diff
@@ -15,7 +15,7 @@ export function useTrialStatus() {
   }
 
   const extendTrial = (days) => {
-    const newEnd = addDays(trialInfo.endsAt, days)
+    const newEnd = addDays(Date.now(), days)
```

Comment:
Weird that this showed up in this PR. Makes me think your branch isn't up to date with master

Reply:
This is showing up now.. branch is hopefully up to date now